src/recursion.js

```diff
@@ -70,8 +70,8 @@ var modulo = function(x, y) {
 var multiply = function(x, y) {
 };
 
-// 13. Write a function that divides two numbers without using the / operator or
-// Math methods.
+// 13. Write a funct  ion that divides two numbers without using the / operator or
```

Comment:
I like what you've done with the word "function" and spacing.

Reply:
DOH! Fixed in second commit.